BITCORNService/Utils/ReferralUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BITCORNService.Models;
+using BITCORNService.Utils.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace BITCORNService.Utils
+{
+    public static class ReferralUtils
+    {
+        public static void UpdateReferralSync(BitcornContext dbContext, PlatformId platformId)
+        {
+            int? userId =0;

```

Comment:
this functionality already exists in util
`int? userId = BitcornUtils.GetUserForPlatform(platformId, dbContext).Select(u => u.UserId).FirstOrDefault();`